master/internal/kubernetes/pods.go

```diff
@@ -109,6 +120,11 @@ func (p *pods) Receive(ctx *actor.Context) error {
 	case sproto.StopPod:
 		p.receiveStopPod(ctx, msg)
 
+	case resourceDeletionFailed:
+		if msg.err != nil {
+			return errors.Wrap(msg.err, "error deleting leftover kubernetes resource")
```

Comment:
nonblocking: This probably happens if k8s API fails to delete, which shouldn't fail the `pods` actor.